Nuxt ssr catalogue fails to start

**What version of EMX2 are you using (see footer)**
8.155.1 or 8.156.0 ( does work on 8.155.0 )

**Describe the bug**
Node.js v18.13.0
node:internal/errors:491
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/.output/server/node_modules/node-fetch-native/dist/polyfill.mjs' imported from /.output/server/index.mjs
    at new NodeError (node:internal/errors:400:5)
    at finalizeResolution (node:internal/modules/esm/resolve:326:11)
    at moduleResolve (node:internal/modules/esm/resolve:945:10)
    at defaultResolve (node:internal/modules/esm/resolve:1153:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:842:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
    at link (node:internal/modules/esm/module_job:76:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}

**To Reproduce**
Steps to reproduce the behavior:
1. run service via docker file

--- a/apps/nuxt3-ssr/package.json
+++ b/apps/nuxt3-ssr/package.json
@@ -20,7 +20,7 @@
     "@tailwindcss/typography": "^0.5.8",
     "autoprefixer": "^10.4.13",
     "graphql-tag": "2.12.6",
-    "nuxt": "3.2.2",
+    "nuxt": "3.2.0",
     "nuxt-proxy": "^0.4.0",
     "postcss": "^8.4.19",
     "postcss-custom-properties": "^13.0.0",

